backend/src/ml_space_lambda/group/lambda_functions.py

```diff
@@ -53,17 +53,6 @@ def _add_group_user(group_name: str, username: str, permissions: Optional[List[P
         raise e
 
 
-def _total_group_owners(group_name: str):
```

Comment:
Just to make sure I'm understanding correctly - all Admins are basically owners for all groups, right? Just making sure we can never have a situation where an Admin removes themself from a Group as a member and now we have empty groups floating around that nobody can see in the UI.

Reply:
Correct, admins can view all groups all the time